cli/running/running.go

```diff
@@ -225,6 +323,18 @@ func createPIDFile(pidFileName string) error {
 	//    user:   read/write
 	//    group:  read
 	//    others: read
+	pidAbsDir := filepath.Dir(pidFileName)
+	if _, err := os.Stat(pidAbsDir); err != nil {
+		if os.IsNotExist(err) {
+			err = os.MkdirAll(pidAbsDir, 0770)
```

Comment:
Don't we want to make default dir permissions a constant?

Reply:
done